packages/cross-chain-data-bridge/src/services/envVars.ts

```diff
@@ -0,0 +1,43 @@
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+export const getEnvironmentV2 = (): string => {
```

Comment:
can we reuse these from the commons package?

Reply:
I think a lot of functions should be reused, but at the moment all packages reimplement these functions